src/wrappers/express.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable prefer-rest-params */
+/**
+ * @fileoverview Handlers for Express instrumentation
+ */
+
+const shimmer = require('shimmer');
+const tryRequire = require('../try_require.js');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const traceContext = require('../trace_context.js');
+const expressRunner = require('../runners/express.js');
+
+const express = tryRequire('express');
+
+
+/**
+ * Express requests middleware
+ * @param {Request} req The Express's request data
+ * @param {Response} res The Express's response data
+ * @param {Function} next express function
+ */
+function expressMiddleware(req, res, next) {
+    // Initialize tracer
+    const tracerObj = tracer.createTracer();
+    tracer.traceGetter = traceContext.getTracer;
+    tracer.restart(tracerObj);
+    let expressEvent;
+    const startTime = Date.now();
+    try {
+        expressEvent = expressRunner.createRunner(req, startTime);
+        tracer.addRunner(expressEvent, undefined, tracerObj);
+    } catch (err) {
+        utils.debugLog(err);
+    }
+
+    // Inject trace functions
+    req.epsagon = {
+        label: tracer.label,
```

Comment:
use object destructing (also in setError)

Reply:
Done